src/calculateRentalCost.test.js

```diff
@@ -3,21 +3,17 @@
 const calculateRentalCost = require('./calculateRentalCost');
 
 test(`Do not add any discount for less than 3 days`, () => {
-  expect(calculateRentalCost(2))
-    .toBe(80);
+  expect(calculateRentalCost(2)).toBe(80);
 });
 
 test('Should add the basic discount from 3 to 6 days of rent', () => {
-  expect(calculateRentalCost(3))
-    .toBe(100);
+  expect(calculateRentalCost(3)).toBe(100);
 });
 
 test('Should add the basic discount from 3 to 6 days of rent', () => {
-  expect(calculateRentalCost(6))
-    .toBe(220);
+  expect(calculateRentalCost(6)).toBe(220);
```

Comment:
Confirm that the expected value of 220 is correct for 6 days of rent. According to the task description, renting for 3 or more days should provide a $20 discount. If the daily rate is $40, then the expected cost for 6 days could be (6 * $40) - $20 = $220. If the implementation differs, this test case needs to be updated to match the correct expected value.